src/components/ResourceCard/AddResource/ResourceSelectActivity.js

```diff
@@ -127,39 +124,15 @@ let ResourceSelectActivity = (props) => {
                       <FontAwesomeIcon icon="chevron-left" />
                       Back
                     </div>
-                    Select the activity you want to build from the options below:
+                    Select the activity you want to build from the options
+                    below:
```

Comment:
Why do we need this change? Is that also from Prettier?

Reply:
prettier removed